cumulus_etl/fhir/fhir_client.py

```diff
@@ -259,8 +262,8 @@ def create_fhir_client_for_cli(
             # Use the input URL as the base URL. But note that it may not be the server root.
             # For example, it may be a Group export URL. Let's try to find the actual root.
             client_base_url = root_input.path
-            client_base_url = re.sub(r"/Patient/?$", "/", client_base_url)
-            client_base_url = re.sub(r"/Group/[^/]+/?$", "/", client_base_url)
+            client_base_url = re.sub(r"/Patient($|/.*)", "/", client_base_url)
+            client_base_url = re.sub(r"/Group/[^/]+($|/.*)", "/", client_base_url)
```

Comment:
Unrelated, but I discovered that now that we support input URLs with arguments like `/$export?_type=xxx`, this regex that tries to find the root for authentication purposes needed to handle that better.